docker-compose.yml

```diff
@@ -2,8 +2,6 @@ version: '2'
 services:
   web:
     build: .
-    command: java -Dserver.port=4567 -jar ./build/libs/bt-example-0.1.0.jar
```

Comment:
I'm not sure why this jar was ever being used here, it's the default jar task output (skinny jar) and won't work without jamming the dependencies into the dockerfile or something.  The new jar is the nice and shiny fat/uber jar the springboot plugin gives us, and it's good to run with no fuss.

Reply:
yeha that decision is lost to time